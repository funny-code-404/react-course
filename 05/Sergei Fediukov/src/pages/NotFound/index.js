const NotFound = () => {
    return (
        <div>
            <h1>
                page doesn't exist
           </h1>
        </div>
    )
}
export default NotFound