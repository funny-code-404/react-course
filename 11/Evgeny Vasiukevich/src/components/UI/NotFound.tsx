import React from 'react';
import styled from 'styled-components';
import { getFontsFragment } from '../../utils/utils';

type Props = {
    title: string,
}

const NotFound = ({title}: Props) => {
    return (
        <StyledNotFound>
            <h1>{ title }</h1>
        </StyledNotFound>
    );
};

const StyledNotFound = styled.div`
    width: 100%;

    & > h1 {
        ${getFontsFragment('lg')}
        text-align: center;
    }
`

export default NotFound;