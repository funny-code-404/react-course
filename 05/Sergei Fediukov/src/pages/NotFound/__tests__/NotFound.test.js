import { configure, shallow } from 'enzyme'
import Adapter from '@wojtekmaj/enzyme-adapter-react-17'

import NotFound from '..'


configure({ adapter: new Adapter() });

describe('Should render correctly "NotFound"', () => {

    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<NotFound />)
    })
    it('to Match Snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })
    it('component is defined', () => {
        expect(wrapper).toHaveLength(1)
    })

})