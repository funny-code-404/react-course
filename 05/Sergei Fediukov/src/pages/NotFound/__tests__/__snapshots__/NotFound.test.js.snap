// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly "NotFound" to Match Snapshot 1`] = `
<div>
  <h1>
    page doesn't exist
  </h1>
</div>
`;
